package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "devDependencies": {
+    "@stoplight/spectral-cli": "^6.2.0",
```

Comment:
I think, given we've got dependabot etc, I'd make this a fixed dependency and lose the `^`, not too strongly opinionated here though

Reply:
Sure

Reply:
do we need to configure dependabot for package.json?

Reply:
looks like we may not need package at all I hope